src/v2/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -183,14 +194,12 @@ export class ShippingRoute extends Component<ShippingProps, ShippingState> {
         ? address
         : convertShippingAddressForExchange(
             // @ts-expect-error STRICT_NULL_CHECK
-            this.getAddressList()[parseInt(this.state.selectedSavedAddress)]
-              .node
+            this.getAddressList()[parseInt(selectedSavedAddress)].node
```

Comment:
Does anything bad happen here with `parseInt` when `selectedSavedAddress === NEW_ADDRESS`? L133 sets it to that value.

Reply:
selectedSavedAddress === NEW_ADDRESS, means that need to show a clean Address form when you deleted all your saved addresses.

Reply:
Ah -- now I see that these lines that do `parseInt(selectedSavedAddress)` are all guarded by the check to `this.isCreateNewAddress()`, so they'll never be hit when `selectedSavedAddress === NEW_ADDRESS`. Thanks for clarifying!